addons/payment_razorpay/const.py

```diff
@@ -113,7 +113,7 @@
 
 # The maximum amount in INR that can be paid through an eMandate.
 MANDATE_MAX_AMOUNT = {
-    'card': 500000,
+    'card': 1000000,
```

Comment:
Razorpay does not allow any payment greater then 500k but it allows token `max_amount` to 1M (I was 500k before) so changing it to 1M in case some card might allow upto 1M then token `max_amount` should not prevent 

video https://app.screencastify.com/v3/watch/fKbTpNPxudkKGWvu4VgU